src/renderer/modules/webpack/get-modules.ts

```diff
@@ -22,44 +22,66 @@ export function getExports<T>(m: RawModule): T | undefined {
 }
 
 /**
- * Iterates over an object and its top-level children that could have properties
+ * Iterates over an object and its top-level and second-level (if specified) children that could have properties
```

Comment:
The option should be to make this check in the prototype, not to check a second a second level.

Reply:
okay so that's handled in filter because if we get prototype here then we wouldn't be able to return the parent function for those prototype

Reply:
I'm not saying to return the prototype, I'm saying it should only be checking prototype specifically, not just any second-level object

Reply:
that would need us to pass the props to check here too. while we can already do that in the function where this is used. and even if we add check here this would still be iterator and will add just some more milliseconds imo

Reply:
No you wouldn't

Reply:
well, it already returns only objects and functions. only those can have prototypes anyway so idk what the issue is, I just read the code after some time, I forgot it already did this.

Reply:
https://github.com/Tharki-God/replugged/blob/byPrototype/src/renderer/modules/webpack/get-modules.ts#L48